crates/database/src/events.rs

```diff
@@ -169,8 +171,25 @@ async fn insert_presignature(
     Ok(())
 }
 
+async fn insert_block(
+    ex: &mut PgConnection,
+    block_number: i64,
+    block_hash: BlockHash,
```

Comment:
In a separate function we might also want the ability to delete old blocks from before the reorg limit so that we don't store blocks forever. This would be called as part of maintenance. Or by EventHandler.

Reply:
Removed code change.